cli/command/cli.go

```diff
@@ -121,6 +123,12 @@ func (cli *DockerCli) ClientInfo() ClientInfo {
 	return cli.clientInfo
 }
 
+// ContentTrustEnabled returns if content trust has been enabled by an
```

Comment:
nit: reads a bit odd, perhaps could be `returns whether trust has been enabled..`